vagrant/scripts/build-zookeeper.sh

```diff
@@ -2,13 +2,13 @@
 
 cd $(mktemp -d)
 
-yum -y install java
-
-wget http://apache.org/dist/zookeeper/zookeeper-3.4.9/zookeeper-3.4.9.tar.gz
+if [ ! -e /etc/yum.repos.d/cloudera-cdh5.repo ]; then
+    wget https://archive.cloudera.com/cdh5/redhat/5/x86_64/cdh/cloudera-cdh5.repo?_ga=2.172934241.314812559.1496985621-1968320782.1496291714 -O /etc/yum.repos.d/cloudera-cdh5.repo
+fi
 
-tar xvf zookeeper-3.4.9.tar.gz -C /opt
+yum -y install zookeeper
 
-cd /opt/zookeeper-3.4.9/conf
+cd /etc/zookeeper/conf
```

Comment:
where will it install zookeeper? cause /opt/zookeeper-3.4.9/bin is in vagrant_aliases.py and other startup scripts. 

why is cloudera preferable to using stock zk jars?